src/pages/teacher/index.vue

```diff
@@ -8,7 +8,8 @@
     <TabMenu
       :tabs="[
         { name: t('my-classes'), background: '#2E9DF9', id:'classes', color: 'white' },
-        { spacer: true, width: 1 },
+        // { spacer: true, width: 1 },
+        { name: t('dashboard') === 'no slug dashboard' ? 'DASHBOARD' : t('dashboard'), background: '#1E6DB8', id:'dashboard', color: 'white' },
```

Comment:
Lets add our first slug! We can define the dashboard slug on the real translation service